### PR TITLE
Table component

### PR DESCRIPTION
**Pull request by ulrikus**
# What?
Makes a component that takes in an array of `TableElement` comprised of a title and a detail styled by its separate class.
The different table components are:
- `TextTableComponent`, takes in two strings
- `DateTableComponent`, takes in a String, a Date, a Locale and a TimeZone

May need to support the following types (eventually):
- Area
- Volume
- Price

### CODE REVIEW

File: Components/ObjectViewController.swift [hunk 1]
```diff
@@ -14,13 +14,21 @@ class ObjectViewController: UIViewController {
         return attributedString
     }()
 
+    let tableComponents: [TableRowModel] = {
+        let currency = "kroner"
+        return [
+            PriceTableComponent(title: "Prisantydning", price: 2500000, currency: currency),
```
Comment (3lvis) [inline Components/ObjectViewController.swift]: We should focus on the torget elements I don't think we have a price table component there 🤔

Comment (ulrikus) [reply Components/ObjectViewController.swift]: That's true. I'll continue with torget specific stuff then.

File: Components/ViewComponents/TableComponent/PriceTableComponent/PriceTableComponentView.swift [hunk 2]
```diff
@@ -47,9 +55,13 @@ public class PriceTableComponentView: UIView {
...
                 return
             }
+            priceFormatter.locale = component.locale
+            priceFormatter.maximumSignificantDigits = String(component.price).count
+            accessibilityPriceFormatter.locale = component.locale
+            accessibilityPriceFormatter.maximumSignificantDigits = String(component.price).count
             titleLabel.text = component.title
             detailLabel.text = priceFormatter.string(from: component.price as NSNumber)! + ",-"
-            accessibilityLabel = component.title + ": " + String(component.price) + component.currency
+            accessibilityLabel = component.title + ": " + accessibilityPriceFormatter.string(from: component.price as NSNumber)! + currency
```
Comment (3lvis) [inline Components/ViewComponents/TableComponent/PriceTableComponent/PriceTableComponentView.swift]: Let's avoid forced unwrapping 💣

**Review by 3lvis — CHANGES_REQUESTED** [review]

File: Components/ObjectViewController.swift [hunk 3]
```diff
@@ -14,13 +14,30 @@ class ObjectViewController: UIViewController {
...
+            PriceTableComponent(title: "Prisantydning", price: 2500000, locale: locale),
+            PriceTableComponent(title: "Formuesverdi", price: 500000, locale: locale),
+            PriceTableComponent(title: "Kommunale avgifter", price: 2100, locale: locale),
+        ]
+    }()
+
+    let textTableComponents: [TableRowModel] = {
+        return [
+            TextTableComponent(title: "FINN-kode", detail: "123456789"),
+            TextTableComponent(title: "Sist endret", detail: "25. nov 2017, 08:00"),
```
Comment (3lvis) [inline Components/ObjectViewController.swift]: This should be a DataTableComponent.

**Review by 3lvis — DISMISSED** [review]

File: Components/ObjectViewController.swift (hunk 3 above)
Comment (ulrikus) [reply Components/ObjectViewController.swift]: Do you mean Date* for the last component?

Comment (3lvis) [reply Components/ObjectViewController.swift]: Yes! :dolan:

**Comment by 3lvis** [comment]
I think we should either figure out a way of using the PriceComponent instead of duplicating the logic in the PriceTableComponent, or we should remove the PriceTableComponent.

**Review by 3lvis — APPROVED** [review]
Nice work, Ulrik!

**Pull request merged**